src/components/app/list-new/HelmAppList.tsx

```diff
@@ -364,95 +416,101 @@ export default function HelmAppList({ serverMode, payloadParsedFromUrl, sortAppl
         );
     }
 
-    function renderAllCheckModal(){
-        return <div style={{ width: '600px', margin: 'auto', marginTop: '20px' }} className="bcn-0 pt-20 pb-20 pl-20 pr-20 br-8 en-1 bw-1 mt-20">
-            <AllCheckModal />
-        </div>
+    function renderAllCheckModal() {
+        return (
+            <div
+                style={{ width: '600px', margin: 'auto', marginTop: '20px' }}
+                className="bcn-0 pt-20 pb-20 pl-20 pr-20 br-8 en-1 bw-1 mt-20"
+            >
+                <AllCheckModal />
+            </div>
+        );
     }
 
-    function askToSelectClusterId(){
-        return <div style={{ height: "calc(100vh - 150px)" }}>
-            <EmptyState>
-                <img src={NoClusterSelectImage} width="250" height="250" alt="No Cluster Selected" />
-                <h2 className="fs-16 fw-4 c-9">Select cluster to see deployed apps</h2>
-                <p className="text-left" style={{width: '300px'}}>Helm-based applications deployed from devtron or other sources will be shown here. Learn more</p>
-            </EmptyState>
-        </div>
+    function askToSelectClusterId() {
+        return (
+            <div style={{ height: 'calc(100vh - 150px)' }}>
+                <EmptyState>
+                    <img src={NoClusterSelectImage} width="250" height="250" alt="No Cluster Selected" />
+                    <h2 className="fs-16 fw-4 c-9">Select cluster to see deployed apps</h2>
+                    <p className="text-left" style={{ width: '300px' }}>
+                        Helm-based applications deployed from devtron or other sources will be shown here. Learn more
+                    </p>
+                </EmptyState>
+            </div>
+        );
     }
 
-    function askToClearFilters(){
-        return <div style={{height : "calc(100vh - 250px)"}}>
-        <Empty
-            view={AppListViewType.NO_RESULT}
-            title={'No apps found'}
-            message={"We couldn't find any matching applications."}
-            buttonLabel={'Clear filters'}
-            clickHandler={clearAllFilters}
-        />
-        </div>
-    }
-    
-    function askToClearFiltersWithSelectClusterTip (){
-        return <div className="flex column">
-            {askToClearFilters()}
-            <div style={{width : '300px'}}>
-                <p className="bcb-1 cn-9 fs-13 pt-10 pb-10 pl-16 pr-16 eb-2 bw-1 br-4 cluster-tip flex left top">
-                    <div><InfoFill className="icon-dim-20" /></div>
-                    <div className="ml-12">
-                        <span className="fw-6">Tip </span>
-                        <span>Select a cluster from above filters to see apps deployed from outside devtron.</span>
+    function askToClearFilters(showTippy?: boolean) {
```

Comment:
boolean showTippy name is not correct. 
change to -> showTipToSelectCluster

Reply:
done